docker-compose.yml

```diff
@@ -1,25 +1,8 @@
 version: '3'
 
 services:
-  endpoint_querier:
-    build: 
-      context: ./networkstatsquerier
-    depends_on:
-        - lantern-mq
-    restart: on-failure
-    environment:
-        - LANTERN_QUSER=${LANTERN_QUSER}
-        - LANTERN_QPASSWORD=${LANTERN_QPASSWORD}
-        - LANTERN_QHOST=${LANTERN_QHOST}
-        - LANTERN_QPORT=${LANTERN_QPORT}
-        - LANTERN_ENDPTINFO_NETSTATS_QNAME=${LANTERN_ENDPTINFO_NETSTATS_QNAME}
-        - LANTERN_ENDPTQRY_NUMWORKERS=${LANTERN_ENDPTQRY_NUMWORKERS}
-    volumes:
-        - ./scripts/wait-for-it.sh:/etc/lantern/wait-for-it.sh
-    command: /etc/lantern/wait-for-it.sh lantern-mq:5672 -- ./main
-
   pg_prometheus:
```

Comment:
docker image name should be changed

Reply:
There are other instances where lantern_dbhost = pg_prometheus that will need to be changed as well